packages/modelviewer.dev/examples/tester.html

```diff
@@ -105,7 +105,7 @@
     <h1>
       <a href="https://github.com/GoogleWebComponents/model-viewer" target="_blank">&lt;model-viewer&gt;</a>
     </h1>
-    Drop a <a href="https://github.com/KhronosGroup/glTF" target="_blank">GLB or zipped GLTF</a> for the model =><br/>
+    Drop a <a href="https://github.com/KhronosGroup/glTF" target="_blank">glB or glTF</a> for the model =><br/>
```

Comment:
"Embedded glTF" maybe?
If a user drops a gltf that depends on bin files it won't work.

Reply:
With the updates in this PR, it `.bin` files should work too. See the `simple-dropzone` dependency: https://github.com/GoogleWebComponents/model-viewer/pull/941/files#diff-4adb71782bb4c74dc2a63a7d1f3a533aR22-R23

Reply:
Oh I see!